docs/course-information.md

```diff
@@ -7,43 +7,51 @@ description: >-
 
-There will be two types of assignments. We will have smaller lab work that will help you to get familiar with the presented software engineering tools
+There are two types of assignments. We have smaller weekly lab work, which helps you to get familiar with the presented software engineering concepts and tools.
+
+
+Parallel to the lab work, you work on an individual challenge, where you apply the learned concepts and tools with the ultimate goal to contribute to a large-scale community simulation software package. The challenge is structured in three parts, whereas each part is completed by a short presentation of the intermediate results in class:
+
+1. You get acquainted with the basic functionality of a large-scale simulation software package (such as FEniCS, PETSc, TRILINOS, or SU2) by studying tutorials and documentation (first quarter of the course)
+2. You analyze the RSE infrastructure and the development cycle of the software package (second quarter of the course). 
+3. You contribute to the software package. The contribution can be small, but should not be trivial. Possible examples: Adding a new tutorial, extending the documentation, working on a "good first issue", adding support of a new package manager. Important is to properly go through all development steps if possible (contact community, open issue, open pull request, test, review, merge).
 
-Parallel to the lab work you will get a bigger assignment where you work with a bigger open-source project that is focusing on simulations or some aspect of simulations. Some of these steps:
 
-1. Get familiar with a certain open-source project related to simulations.
-2. Analyze the software engineering infrastructure of this project.
-3. You contribute to the project by creating a pull request.
+The lab work and assignments are an important part of the course. There will be **no** written or oral exam at the end of the lecture. Instead, you are evaluated continuously based on the lab work and assignments.
 
-Each of these steps incorporates the presentation of your findings and work in class.
+## Additional information for SimTech students
 
-The lab work and assignments are an important part of this course. There will be **no** written or oral exam at the end of the lecture. Instead, you will be evaluated continuously based on the lab work and assignments.
+You might also be interested in the course _sustainable development of simulation software_ by Bernd Flemisch and Dennis Gläser, which is starting in summer term 2022. Their course has a stronger focus on software engineering patterns and project work in C++ for simulation software. Our course "Simulation Software Engineering" focuses more on the tools commonly used in the development of simulation software. The course of Flemisch/Gläser is **not** a prerequisite to follow our course on simulation software engineering and neither vice versa. We encourage you, however, to follow both courses if you are interested in the topic.
```

Comment:
- Follow capitalization of Bernd's and Dennis' course name, e.g. "Sustainable development of simulation software"? I am not sure if this is the actual capitalization used.
- Why is their course name emphasized using underscores and our course's name is in quotation marks?
- We should maybe agree on some Markdown code style. I prefer the asterisk for emphasizing while you use underscores. At least when it is about mild emphasis (italic).

Reply:
I made both formats consistent: https://github.com/Simulation-Software-Engineering/homepage/pull/2/commits/944a12e167c6d0edf95297a80fb15e6a7729ee5e

Markdownlint: yes :+1:
I personally use underscores for italic and (single?) asterisks for bold. How I use it there is a difference in emphasis. Italic is meant to make the sentence easier to read. Bold means to attract the reader's attention to this sentence / paragraph. 
I am happy to compromise on a common standard however :grin: 
... not sure if we can _lint_ such details.

Reply:
Single asterisk is italic and double asterisk is bold so the number of asterisks  "scales" with increased attention. That is why tend to use the asterisks. I think I also stumbled upon https://www.markdownguide.org/basic-syntax/#emphasis some time which votes against the underscores. 

I will configure some markdown lint CI into the repository (especially for the PRs) and then we can figure out the proper configuration over time. I have not used it yet so I am not sure whether we can (or want to) be so picky on the details.

Reply:
You convinced me :blush: 
Changed to asterisks.